rumqttd/src/router/routing.rs

```diff
@@ -920,6 +980,7 @@ impl Router {
 
         let connection = &mut self.connections[id];
         let broker_topic_aliases = &mut connection.broker_topic_aliases;
+        let mut skipped_requests = VecDeque::new();
```

Comment:
What if we place this in the router stuct, since it will only ever be one consume running we can have it there and make sure to clear it. Also make it initialized with a capacity to not have to allocate everytime. There is a max amount it can contain right?

Reply:
Yup, thought about it, but then I said, call to `::new()` doesn't allocate unless elements are pushed into it right? So, will this be a premature/unnecessary optimization?

Reply:
calling new() seems better choice, bringing it back in consume.